examples/3d/anti_aliasing.rs

```diff
@@ -141,7 +148,7 @@ fn modify_aa(
         camera
             .remove::<Fxaa>()
             .remove::<Smaa>()
-            .insert(TemporalAntiAliasBundle::default());
+            .insert(TaaComponents::default());
```

Comment:
The insert doesn't need to use the bundle since the components are required anyway. Also would be nice if these `removes` got batched but that's not really a new issue in this PR